var/spack/repos/builtin/packages/glib/package.py

```diff
@@ -28,6 +28,8 @@ class Glib(MesonPackage, AutotoolsPackage):
 
     license("LGPL-2.1-or-later")
 
+    version("2.80.2", sha256="b9cfb6f7a5bd5b31238fd5d56df226b2dda5ea37611475bf89f6a0f9400fe8bd")
+    version("2.79.3", sha256="00d1a313677ed5f5c5e6f7a77ca66e31cf7aa1e3d6b92b6041a16d7a024580c6")
```

Comment:
2.79 is a development version, let's not add this one.

Reply:
```suggestion
```